gym/f110_gym/envs/dynamic_models.py

```diff
@@ -24,7 +24,7 @@
 Track dynamic model
 Following the implementation of commanroad's Single Track Dynamics model
 Original implementation: https://gitlab.lrz.de/tum-cps/commonroad-vehicle-models/
-Author: Hongrui Zheng
+Author: Hongrui Zheng, Renukanandan Tumu
```

Comment:
Nothing against. 
Shall we put credits in each file or remove them from all files and update the license?

Reply:
I agree, we should either burn all of them or update all of them. Git blame exists, so I'm in support of burning all of them.

Reply:
Move discussion to next meeting